Readme.md

```diff
@@ -30,6 +30,16 @@ user.reset_delta_changes!
 user.delta_changes # => {}
 ```
 
+Testing
+======
+
+To run tests with Rails 4.2: `$ rake spec`
```

Comment:
ideally the Gemfile should be the same as latest ... so `eval_gemfile Dir["gemfiles/*.gemfile"].sort.last`